packages/datadog-instrumentations/src/cucumber.js

```diff
@@ -85,6 +85,7 @@ function wrapRun (pl, isLatestVersion) {
       return promise
     } catch (err) {
       errorCh.publish(err)
+      throw err
```

Comment:
yeah I don't know how to test it, since I think we'd have to provoke an internal error in cucumber. But if we don't throw in here, we'd be modifying how cucumber behaves, wouldn't we?